app/assets/javascripts/admin/addon/lib/site-setting-matcher.js

```diff
@@ -0,0 +1,78 @@
+export default class SiteSettingMatcher {
+  constructor(filter, siteSetting) {
+    this.filter = filter;
+    this.siteSetting = siteSetting;
+    this.strippedQuery = filter.replace(/[^a-z0-9]/gi, "");
+    this.fuzzyRegex = new RegExp(this.strippedQuery.split("").join(".*"), "i");
+    this.fuzzyRegexGaps = new RegExp(
+      ".*" + this.strippedQuery.split("").join("(.*)"),
+      "i"
+    );
+    this.matchStrength = 0;
+  }
+
+  get isNameMatch() {
+    const name = this.siteSetting.get("setting").toLowerCase();
```

Comment:
I think you can do `siteSetting.setting` since you don't need this to be "reactive" in ember